src/task/index.html

```diff
@@ -106,6 +106,38 @@
             <option value="ASSIGNED">Assigned</option>
           </select>
         </div>
+        <div class="inputBox">
+          <label for="taskLevel">Task Level</label>
+          <br />
+          <div class="inputBox">
+            <label for="level" class="editable">
+              Category:<span class="required">*</span>
+            </label>
+            <select
+              required
+              class="input notEditing"
+              id="category"
+              name="category"
+            >
+              <option selected value="system_design">System Design</option>
+              <option selected value="frontend">Frontend</option>
```

Comment:
Why are both fields selected?
Also this dropdown currently have only 4 fields, how will we add `other` skill? Suppose a custom skill like `React`